ui/pom.xml

```diff
@@ -84,6 +70,36 @@
             <version>1.1</version>
         </dependency>
         <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>script-security</artifactId>
+            <version>1.18</version>
```

Comment:
From #7; sometimes you need to override an overly old dependency from the wrong path in the tree. (Whether the above is actually new enough, I am not sure.)